components/brave_vpn/browser/BUILD.gn

```diff
@@ -54,7 +54,7 @@ source_set("unit_tests") {
     "//base",
     "//brave/components/brave_vpn/browser/connection:api",
     "//brave/components/brave_vpn/browser/connection:common",
-    "//brave/components/brave_vpn/browser/connection:sim",
+    "//brave/components/brave_vpn/browser/connection/ikev2:sim",
```

Comment:
It seems android CI assertion failure with this.
it should be in `!is_android`?

Reply:
yes